Maui.RevenueCat.InAppBilling/Platforms/iOS/RevenueCatBillingiOS.cs

```diff
@@ -34,6 +34,33 @@ public partial class RevenueCatBilling : IRevenueCatBilling
             throw;
         }
     }
+
+    public async partial Task<Dictionary<string, IntroElegibilityStatus>> CheckTrialOrIntroDiscountEligibility(IList<string> identifiers, CancellationToken cancellationToken)
+    {
+        try
+        {
+            using var eligibilities = await _purchases.CheckTrialOrIntroDiscountEligibilityAsync(identifiers);
+            if (eligibilities is null)
+            {
+                return new();
+            }
```

Comment:
As written empty worked fine by just returning an empty dictionary, since it went through the loop zero times.  Changed it anyway though it is more consistent with the other methods.